kadi/events/models.py

```diff
@@ -1466,23 +1481,30 @@ def get_target_attitudes(cls, event, msidset):
 
     @classmethod
     @import_ska
-    def get_one_shot(cls, guide_start, one_shot_msid):
+    def get_one_shot(cls, guide_start, aux_msidset):
         """
         Define one_shot attribute (one-shot attitude update following maneuver)
         """
-        # No acq => guide transition so no one-shot defined.  Use 0.0 as a convenience.
+        # No acq => guide transition so no one-shot defined.  Use -99.0 as a convenience for
+        # the one shot length, -999.0 for the axis values.
         if guide_start is None:
-            return -99.0
-
-        t0 = DateTime(guide_start).secs
-        n_times = len(one_shot_msid.times)
-        i0, i1 = np.searchsorted(one_shot_msid.times, [t0 - 10, t0 + 10])
-        if i0 == 0 or i1 == 0 or i0 == n_times or i1 == n_times:
-            raise ValueError('Unable to find complete data for one-shot attitude value'
-                             ' {} {} {} {} {}'
-                             .format(i0, i1, t0, np.min(one_shot_msid.times),
-                                     np.max(one_shot_msid.times)))
-        return np.max(one_shot_msid.vals[i0:i1])
+            return {'one_shot': -99.0,
+                    'one_shot_roll': -999.0, 'one_shot_pitch': -999.0, 'one_shot_yaw': -999.0}
+        # Save the first post maneuver / post ACQ attitude error sample for each AOATTER axis
+        pm_att_err = {}
+        for ax in [1, 2, 3]:
+            msid = aux_msidset['aoatter{}'.format(ax)]
+            ok = ((msid.times >= DateTime(guide_start).secs)
```

Comment:
For testing you could plot the difference between current and new `one_shot` values for a representative sample (e.g. a year of one shots).  Hopefully those will all be less than a few arcsec, and that would be sufficient testing for merge.